src/ts/contentScripts/dispatcher/index.ts

```diff
@@ -20,6 +20,20 @@ const dispatchMap = new Map([
 
 browser.runtime.onMessage.addListener((command) => dispatchMap.get(command)?.());
 
+document.addEventListener('keydown', ev => {
+    const enter = 'Enter';
+    const isExitingTitle = (ev: KeyboardEvent) => {
+        return ev.target.parentElement instanceof HTMLHeadingElement;
+    }
+    const isNoBlockSelected = (ev: KeyboardEvent) => {
```

Comment:
I think you can use something like `!Roam.getActiveRoamNode()` to check for this

Reply:
Ok. I'll give that a shot…assuming it's even worth continuing with this small feature given your note re: Cmd+Enter. Either way, I'll test it just to learn a little bit more. :)

Reply:
Yeah, not sure if makes sense to include this given `Cmd+Enter` exists. I still appreciate you working on this though, and look forward to you building more things! ;)